src/observable/FromEventObservable.ts

```diff
@@ -78,28 +84,31 @@ export class FromEventObservable<T, R> extends Observable<T> {
    */
   static create<T>(target: EventTargetLike,
                    eventName: string,
-                   selector?: (...args: Array<any>) => T): Observable<T> {
-    return new FromEventObservable(target, eventName, selector);
+                   selector?: (...args: Array<any>) => T,
+                   options?: EventListenerOptions): Observable<T> {
+    return new FromEventObservable(target, eventName, selector, options);
   }
 
   constructor(private sourceObj: EventTargetLike,
               private eventName: string,
-              private selector?: (...args: Array<any>) => T) {
+              private selector?: (...args: Array<any>) => T,
+              private options?: EventListenerOptions) {
     super();
   }
 
   private static setupSubscription<T>(sourceObj: EventTargetLike,
                                       eventName: string,
                                       handler: Function,
-                                      subscriber: Subscriber<T>) {
+                                      subscriber: Subscriber<T>,
+                                      options?: EventListenerOptions) {
     let unsubscribe: () => void;
     if (isNodeList(sourceObj) || isHTMLCollection(sourceObj)) {
       for (let i = 0, len = sourceObj.length; i < len; i++) {
-        FromEventObservable.setupSubscription(sourceObj[i], eventName, handler, subscriber);
+        FromEventObservable.setupSubscription(sourceObj[i], eventName, handler, subscriber, options);
       }
     } else if (isEventTarget(sourceObj)) {
       const source = sourceObj;
-      sourceObj.addEventListener(eventName, <EventListener>handler);
+      sourceObj.addEventListener(eventName, <EventListener>handler, <boolean>options);
```

Comment:
Unfortunately we need to typecast the `options` object (for now) due to `addEventListener` only accepts boolean as third parameter. See: https://github.com/Microsoft/TypeScript/issues/9548